newIDE/app/src/Utils/GDevelopServices/Usage.js

```diff
@@ -114,241 +125,162 @@ export type PlanDetails = {|
   |}>,
 |};
 
+export type SubscriptionPlanPrice = {|
+  id: string,
+  planId: string,
+  period: 'week' | 'month' | 'year',
+  isPerUser?: true,
+  currency: 'EUR' | 'USD',
+  region: 'north' | 'south' | 'everywhere',
+  amountInCents: number,
+  periodCount: number,
+|};
+
+export type SubscriptionPlan = {|
+  id: string,
+  isLegacy: boolean,
+  nameByLocale: MessageByLocale,
+  descriptionByLocale: MessageByLocale,
+  bulletPointsByLocale: Array<MessageByLocale>,
+|};
+
+export type SubscriptionPlanWithPrices = {|
```

Comment:
Looking at the code you seem to be handling both `SubscriptionPlanWithPrices` and `SubscriptionPlan` in some places.
I'm wondering if you should just have one concept of `SubscriptionPlan` which contain prices by default.
Otherwise, it's a bit confusing to have 2 different concepts being manipulated